testnet/stacks-node/src/tests/nakamoto_integrations.rs

```diff
@@ -10309,3 +10350,323 @@ fn clarity_cost_spend_down() {
 
     run_loop_thread.join().unwrap();
 }
+
+/// If we get a flash block -- a sortition in which we win, immediately followed by a different
+/// sortition, make sure we first mine a tenure-change block and then a tenure-extend block.
+#[test]
+#[ignore]
+fn test_tenure_change_and_extend_from_flashblocks() {
```

Comment:
I am prob just misunderstanding the test, but I don't see any confirmation about a tenure change/extend being issued in the correct order in this test. Is the fact the follower boots up and the chain advances proof enough?